lib/migrate.js

```diff
@@ -64,10 +64,10 @@ function validateOptions (options) {
           .description('The port to connect on')
       }))
     }),
-    ssl: Joi.object().keys({
+    ssl: [Joi.boolean(), Joi.object().keys({
```

Comment:
@lirantal I think this should be like bellow:

```
ssl: Joi.alternatives().try(Joi.object(), Joi.boolean()).default(false)
```

If we define the object keys for ssl option in this Joi validation, we would limit the valid options only to those declared here.

Also I think it is best to include `false` as the default so no change is needed in other setups.

I'll update the docs accordingly.